helix-core/src/main/java/org/apache/helix/monitoring/mbeans/InstanceMonitor.java

```diff
@@ -42,23 +43,29 @@
   /**
    * Metric names for instance capacity.
    */
-  public enum InstanceMonitorMetrics {
+  public enum InstanceMonitorMetric {
```

Comment:
Will changing the name of a public enum break backward-compatibility?

Reply:
No. This class was added in a previous commit for adding the MaxCapacityUsageGauge. It is not yet deployed into production.